### PR TITLE
feature/GAL-603# error message resolved

### PR DESCRIPTION
**Pull request by Abdul-NHS**
## Description
https://nhsd-jira.digital.nhs.uk/browse/GAL-603

- [ ] Refactoring (non-breaking change)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Bug fix (non-breaking change which fixes an issue)

## Checklist

- [ ] I am familiar with the [contributing guidelines](../docs/CONTRIBUTING.md)
- [ ] I have followed the code style of the project
- [ ] I have added tests to cover my changes
- [ ] I have updated the documentation accordingly
- [ ] This PR is a result of pair or mob programming

---

## Sensitive Information Declaration

To ensure the utmost confidentiality and protect your and others privacy, we kindly ask you to NOT including [PII (Personal Identifiable Information) / PID (Personal Identifiable Data)](https://digital.nhs.uk/data-and-information/keeping-data-safe-and-benefitting-the-public) or any other sensitive data in this PR (Pull Request) and the codebase changes. We will remove any PR that do contain any sensitive information. We really appreciate your cooperation in this matter.

- [ ] I confirm that neither PII/PID nor sensitive data are included in this PR and the codebase changes.

### CODE REVIEW

File: src/app/views/invitation_summary/InvitationSummaryPage.jsx [hunk 1]
```diff
@@ -60,19 +60,24 @@ export default function InvitationSummaryPage(props) {
...
+                totalToInvite={totalToInvite}
+              />}
           </div>
           <div
             data-testid="main-content"
             id="main-content"
             className="nhsuk-grid-column-two-thirds"
           >
-            {displayErrorInvitationSummary || (noInviteToGenerate > totalToInvite) && (
+            {/* {displayErrorInvitationSummary || (noInviteToGenerate > totalToInvite) && (
```
Comment (Abdul-NHS) [inline src/app/views/invitation_summary/InvitationSummaryPage.jsx:75]: commented out for now as the inset text takes precedence over the errors initially created for the invitation algo ticket errors

**Review by chenglawson1 — DISMISSED** [review]

**Review by Amar-NHS — APPROVED** [review]
Approved!

**Review by NeerjaGoyalNHS — APPROVED** [review]
LGTM

**Comment by Abdul-NHS** [comment]
reopening as was suppose to be merged in, not closed

**Pull request merged**